bash/backup/gfs_backup_manager.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+
+# === Configuration ===
+DAILY_RETENTION_DAYS="8"     # Days to keep daily backups
+WEEKLY_RETENTION_WEEKS="12"  # Weeks to keep weekly archives
+MONTHLY_RETENTION_MONTHS="12" # Months to keep monthly archives
+
+DRY_RUN="1" # 1: Only show what would be done. 0: Actually delete (CAUTION!).
+LOG_FILE="/var/log/lxc_backup_cleanup.log"
```

Comment:
```suggestion
LOG_FILE="/opt/community-scripts/log/lxc_backup_cleanup.log"
```
I know var/log is the typical place for logs, but we try to keep all community-script stuff in one folder.